doc/readthedocs/sphinx/auxiliary-programs/auxiliary-programs.rst

```diff
@@ -2292,17 +2292,17 @@ These are followed by groupings of Monthly temperature data.
      Min Hour      16    24    22    15    23    14    16    16    19    20    13     6  
 
      - Average Hourly Statistics for Opaque Sky Cover %
-     Jan    Feb    Mar    Apr    May    Jun    Jul    Aug    Sep    Oct    Nov    Dec   
-     0:01- 1:00     57    36    41    41    38    28    36    35    38    26    44    36 
-     1:01- 2:00     57    38    44    45    34    26    40    36    46    30    43    35 
-     2:01- 3:00     58    39    43    43    37    27    44    36    51    27    43    37 
-     3:01- 4:00     57    41    49    50    39    31    48    34    57    28    45    37 
-     4:01- 5:00     60    43    45    54    45    39    50    36    62    30    47    35 
-     5:01- 6:00     60    46    46    52    45    36    54    41    57    37    51    33 
-     6:01- 7:00     58    48    49    58    48    35    56    45    52    39    52    32 
-     7:01- 8:00     59    49    46    50    44    31    50    46    52    40    49    38 
-     8:01- 9:00     62    51    50    46    40    29    43    36    47    43    48    43 
-     9:01-10:00     61    53    43    41    37    26    37    32    34    38    43    44 
+                  Jan   Feb   Mar   Apr   May   Jun   Jul   Aug   Sep   Oct   Nov   Dec   
```

Comment:
Is the data actually coming out like this?  I don't really have a strong opinion about adjusting this text, but I wouldn't want to mislead someone who might be parsing the output.